configs/bokeh.json

```diff
@@ -46,7 +139,6 @@
   "custom_settings": {
     "separatorsToIndex": "_",
     "attributesForFaceting": [
-      "lang",
```

Comment:
You will not be able to filter on `lang` if you remove it from the `attributesForFaceting`.

Reply:
I must've misunderstood an earlier comment, I thought the only facet was version and so that removing was appropriate.